tests/assert_news/test_news_validator.py

```diff
@@ -0,0 +1,80 @@
+"""Tests for mbed_tools_ci.assert_news.NewsFileValidator."""
+from unittest import mock, TestCase
+from pyfakefs.fake_filesystem_unittest import Patcher
+
+from mbed_tools_ci.assert_news import (
+    NewsFileValidator,
+    NEWS_FILE_NAME_REGEX,
+)
+
+
+class TestValidateFileName(TestCase):
```

Comment:
Do we have a requirement to support `unittest`? If we just need to support `pytest` we don't need to inherit `TestCase`

Reply:
@madchutney and @acabarbaye were keen on using it - they seem to prefer `self.assert`. Personally I'm not a fan, but I didn't feel strong enough to fight that decision.

Reply:
Okay. I definitely prefer using regular asserts over `self.assert` but don't want to start a war over it :)

Reply:
IKR! 🥊 :trollface: